### PR TITLE
feat: Showing Resources status for integration

### PR DESCRIPTION
**Pull request by manish-agrawal-ai**
# Description

Showing Resource status while installing any integrations or integration has been timed out

## How Has This Been Tested?
- [x] For all kind of integration statuses
## Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates? NO
* [ ] I've updated documentation as required by this PR. NA
* [x] I have performed a self-review of my own code.
* [x] I have commented my code, particularly in hard-to-understand areas.
* [x] I have tested it for all user roles.
* [x] I have added all the required unit/api test cases.

## Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: src/components/v2/appDetails/appDetails.type.ts [hunk 1]
```diff
@@ -357,6 +357,10 @@ export interface AppStatusDetailType {
     close: () => void
     appStreamData: any
     showAppStatusMessage?: boolean
+    title? : string
+    status? : string
```
Comment (shivani170) [inline src/components/v2/appDetails/appDetails.type.ts:361]: Are we using this `status` props anywhere?

File: src/setupProxy.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ module.exports = function (app) {
     app.use(
         '/orchestrator',
         createProxyMiddleware({
-            target: 'http://demo.devtron.info:32080/',
+            target: 'http://172.173.174.169:32080/',
```
Comment (shivani170) [inline src/setupProxy.js:7]: Revert this file

Comment (manish-agrawal-ai) [reply src/setupProxy.js]: done

File: src/components/v2/appDetails/appDetails.type.ts (hunk 1 above)
Comment (manish-agrawal-ai) [reply src/components/v2/appDetails/appDetails.type.ts:361]: yes. in the integration resource status

**Review by sohel-rp — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**